### ISSUE TITLE
Version 9.17.3 is still pulling vulnerable okhttp version

### ISSUE THREAD

**Issue by barvek** [IBM/java-sdk-core#183]
I am pulling sdk-core version 9.17.3 and it is still pulling a vulnerable okhttp 4.9.3 dependency into my project which is vulnerable.

The documentation says its updated to 4.10.0 but in reality its still pulling the vulnerable version.

![image](https://user-images.githubusercontent.com/69261941/183138186-0bb5aacd-a58a-4cff-8eee-a6afeefc790c.png)

**Comment by padamstx** [comment]
I double-checked this within my local copy of the java core project (the most recent code) and v4.10 of okhttp is being referenced and not v4.9.3.   Are you sure you are not explicitly declaring okhttp as a dependency?  Perhaps you are specifying v4.9.3 as an override to what is defined by the java core.

**Comment by barvek** [comment]
No I am not defining it explicitly, I have also cleaned my maven cache (m2 repository) locally but I am still have the same issue. On top of that the Snyk vulnerability assessment tool is also reporting this as vulnerability in my project, hence I am adding the version 4.10.0 manually as no other solution seem to be working for me. I have checked that no other dependency in my `pom.xml` is adding okhttp older version as well.
